articles/client-platforms/angularjs/_includes/_login.md

```diff
@@ -34,7 +34,7 @@ With the `authService` injected in this controller, the `login` method can now b
 <button class="btn btn-primary" ng-click="vm.authService.login()">Log In</button>
 ```
 
-When the user clicks the **Log In** button, the Lock widget will be displayed.
+When the user clicks the **Log In** button, the [Lock widget](/libraries/lock) will be displayed.
```

Comment:
Hey @camimonse , we always use relative URLs for internal `auth0.com/docs` links. For example, if the absolute path to the document is `https://auth0.com/docs/identityproviders`, we use `/identityproviders`. These links will be correctly formatted in the build process.